plugins/decl_hdf5/dataset_op.cxx

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
  * Copyright (C) 2015-2024 Commissariat a l'energie atomique et aux energies alternatives (CEA)
- * Copyright (C) 2021 Institute of Bioorganic Chemistry Polish Academy of Science (PSNC)
+ * Copyright (C) 2022 Institute of Bioorganic Chemistry Polish Academy of Science (PSNC)
```

Comment:
The year of this copyright, it is correct? Do you need add some new copyright due to Yacine?

Reply:
Don't see why PSNC should change. Yacine was CNRS

Reply:
I guess there were contributions by Karol. Let's keep the changes that were made.